gulpfile.js

```diff
@@ -188,7 +189,7 @@ packages.forEach(function(pckg, i) {
         stream = stream.pipe(source(pckg.fileName + '.js'))
             .pipe(streamify(babel({
                 compact: false,
-                presets: [[ '@babel/preset-env', { "useBuiltIns": "usage", "corejs": 2 } ]]
+                presets: [[ '@babel/preset-env', { "useBuiltIns": "entry", "corejs": 2 } ]]
```

Comment:
Was just looking at the [babel docs][1] and honestly I don't know if useBuiltIns is the right option if the polyfill is being injected.... 

[1]: https://babeljs.io/docs/en/babel-preset-env#usebuiltins-entry